packages/web/components/templates/article/ArticleContainer.tsx

```diff
@@ -180,8 +187,13 @@ export function ArticleContainer(props: ArticleContainerProps): JSX.Element {
     fontSize,
     margin: marginOverride ?? props.margin ?? 360,
     lineHeight: lineHeightOverride ?? props.lineHeight ?? 150,
-    fontFamily: fontFamilyOverride ?? props.fontFamily ?? 'inter',
-    readerFontColor: highContrastFont ? theme.colors.readerFontHighContrast.toString() : theme.colors.readerFont.toString(),
+    fontFamily:
+      fontFamilyOverride ??
+      props.fontFamily ??
+      (props.isAppleAppEmbed ? undefined : 'inter'),
```

Comment:
So here you think it gets set to a string named "undefined"?

Reply:
I wonder if "unset" would work

Reply:
yeah because `fontFamilyOverride` could be "undefined". I'll see if "unset" works just to make it less confusing.